src/main/java/Command/TodoCommand.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Creates a new todo task
+ */
+package Command;
+
+import Duke.Storage;
+import Duke.TaskList;
+import Duke.Ui;
+import Tasks.Todo;
+
+public class TodoCommand extends Command{
```

Comment:
oops forgot gap between Command and { haha

Reply:
oops sorry, I missed out on that. Thanks for highlighting it :)